player/lua/select.lua

```diff
@@ -0,0 +1,347 @@
+--[[
+This file is part of mpv.
+
+mpv is free software; you can redistribute it and/or
+modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation; either
+version 2.1 of the License, or (at your option) any later version.
+
+mpv is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public
+License along with mpv.  If not, see <http://www.gnu.org/licenses/>.
+]]
+
+local utils = require "mp.utils"
+local input = require "mp.input"
+
+local function show_error(message)
+    mp.msg.error(message)
+    if mp.get_property_native("vo-configured") then
+        mp.osd_message(message)
+    end
+end
+
+mp.add_forced_key_binding(nil, "select-playlist", function ()
+    local playlist = {}
+    local default_item = 1
+
+    for i, entry in ipairs(mp.get_property_native("playlist")) do
+        playlist[i] = select(2, utils.split_path(entry.filename))
+
+        if entry.playing then
+            default_item = i
+        end
+    end
+
+    if #playlist == 0 then
+        show_error("The playlist is empty.")
+        return
+    end
+
+    input.select({
+        prompt = "Select a playlist entry:",
+        items = playlist,
+        default_item = default_item,
+        submit = function (index)
+            mp.commandv("playlist-play-index", index - 1)
+            input.terminate()
+        end,
+    })
+end)
+
+local function format_track(track)
+    return (track.selected and "●" or "○") ..
+        (track.title and " " .. track.title or "") ..
+        " (" .. (
+            (track.lang and track.lang .. " " or "") ..
+            (track.codec and track.codec .. " " or "") ..
+            (track["demux-w"] and track["demux-w"] .. "x" .. track["demux-h"] .. " " or "") ..
+            (track["demux-fps"] and not track.image and string.format("%.3f", track["demux-fps"]) .. "FPS " or "") ..
+            (track["demux-channel-count"] and track["demux-channel-count"] .. "ch " or "") ..
```

Comment:
```suggestion
            (track["demux-channel-count"] and track["demux-channel-count"] .. " ch " or "") ..
```

Reply:
For audio could also add `codec-profile`, it tells if it is atmos, but if it is too much, no need.